QUARKUS-2489.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-2489 OIDC Back channel logout
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2489
+
+This feature provides Back-Channel Logout capability for OIDC extension.
+
+## Scope of the testing
+
+Backchannel logout applies to `quarkus-oidc` and basically is a standard way to logout the current user from all the applications this user is currently logged in, bypassing the user agent.
+
+We will create a valid token by an authorization flow and proceed with the logout flow.
```

Comment:
Who you gonna call? Quarkus app or KC or ..?

Reply:
We are gonna make a request to an authenticated endpoint that will forward the request to KC, once the login is done we will verify that the session was created and then proceed with the logout. And re-check again that the session was removed.

Reply:
Thanks for explanation.